docs/Docker-install-wind.md

```diff
@@ -0,0 +1,208 @@
+
+        Training workspace is the directory where fAIr will store its training files 
+        for example. ***TODO*** Is this about a new checkpoint after fine-tuning? The name implies that this is for temporary files.
+
+        ```
+        mkdir trainings
+        set TRAINING_WORKSPACE=C:\Users\kshitij\fAIr_install\trainings
+        ```
+
+5. Register your Local setup to OSM ***TODO*** Settings in OSM? "your Local setup" = my local fAIr installation?
+
+    - Go to [OpenStreetMap](https://www.openstreetmap.org/), Log in (If needed: Create an account first.).
+    - Click on your Profile and Hit ```My Settings```
+    - Navigate to ```Oauth2 Applications```
+    - Register a new application 
+    - Check permissions for ```Read user preferences``` and set Redirect URI to be ```http://127.0.0.1:3000/authenticate/```. Give it the name  ```fAIr Dev Local```
+    - You will get ```OSM_CLIENT_ID``` and ```OSM_CLIENT_SECRET```. Copy them. 
+
+6. Create environment files
+ 
+    - Create a file ```.env``` in directory backend with a copy of the content in [docker_sample_env](../backend/docker_sample_env).
+ 
+        ```
+        cd backend
+        copy docker_sample_env .env
+        ```
+
+    - Fill in the details of ```OSM_CLIENT_ID``` and ```OSM_CLIENT_SECRET``` in the .env file and generate a unique key and paste it to ```OSM_SECRET_KEY``` (It can be any value in a setup for development).
+    
+        Leave the other items as they are, unless you have a specific, well-defined need.
+
+    - Create ```.env``` in  directory frontend
+
+        ```
+        cd frontend
+        copy .env_sample .env
+        ```
+
+        There is no need to modify this file in a setup for development.
+    
+7. Build and Run containers 
+
+    In directory fAIr, run the following commands:
+
+    ```
+    docker compose build
+    docker compose up
+    ```
+
+8. Run Migrations
+
+    ***TODO*** What does "migration" mean in this context?
+
+    Grab API container and Open Bash: In another command window, go to directory fAIr and execute: 
+
+    ```
+    docker exec -it api bash
+    ```
+
+    Once bash is promoted run the following commands, one at a time: ***TODO*** How do we know when bash is promoted? What does "promoted" mean in this context?
+
+        python manage.py makemigrations
+        python manage.py makemigrations login
+        python manage.py makemigrations core
+        python manage.py migrate
+
+9. Play 
+
+    Restart containers 
+
+    ```
+    docker compose restart
+    ```
+
+    Frontend will be available on port 5000, Backend on 8000, and Flower on 5500. 
+
+    To use your local fAIr installation, go to [Local fAIr](http://127.0.0.1:3000) with your web browser.
+    ***TODO*** There is a list of port numbers, but it doesn't include 3000, which seems to be the most important one. Is this a bug or a feature?
+
+Extra: Want to run your local tiles? 
+
+***TODO*** What is this? What does it do?
```

Comment:
Those instructions are mainly for running local tiles in your computer from local .geotiff without uploading to openaerialmap

Reply:
In my view there is a problem with this part of the instruction. But evidently it would be too difficult to solve. My suggestion is to remove the TODO comment.